example-cucumber4/src/test/java/com/epam/reportportal/example/cucumber4/logging/ParametersTest.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.reportportal.example.cucumber4.logging;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
+ */
+public class ParametersTest {
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(ParametersTest.class);
+	private int itemsCount;
+
+	@Given("^I have (\\d+) (.*) in my pocket$")
```

Comment:
There is also different notation for cucumber which is more popular now, please also add it:
`I have <int> <string> in my pocket`

Reply:
done